common/util.go

```diff
@@ -75,6 +75,9 @@ var (
 	ErrProfEncoder  = fmt.Errorf("unknown VideoProfile encoder for protobufs")
 	ErrProfName     = fmt.Errorf("unknown VideoProfile profile name")
 
+	ErrUnsupportedAudioFormat   = fmt.Errorf("audio format unsupported")
```

Comment:
@eliteprox this is a very usefull error which should be implemented but currently is not used. I tried inputting the following unsupported audio OGG filetype:

[25eb6f27.zip](https://github.com/user-attachments/files/16227789/25eb6f27.zip)

But it threw a audio duration calculation error. Can we implement a audio file type unsupported error?

Reply:
Checking file extensions isn't super reliable and ffmpeg has some edge cases where it returns 0 duration for supported file types, so we currently just check if the duration is <= 0.

Reply:
The status code returned from `GetCodecInfoBytes` (which currently ignored) should indicate something that indicates the file format is unsupported, although I don't know exactly what the code would be. However I am unsure why the API returns both a code and an error; ideally it should be one or the other.

For what it is worth in this specific case, ffmpeg should be able to detect the duration for this file.  Ogg is probably just not compiled into in the production LPMS build, which has a slimmed-down version of ffmpeg. Adding the ogg demuxer to `install_ffmpeg.sh` might be enough for this case.